status/status.go

```diff
@@ -46,6 +47,29 @@ func main() {
 			Value: 0,
 			Usage: "debug-level: `integer`: 1 for terse, 5 for maximal",
 		},
+		cli.BoolFlag{
+			Name: "connectivity, c",
+		},
+	}
+	app.Commands = cli.Commands{
+		{
+			Name:      "connectivity",
```

Comment:
This reminds me https://youtu.be/eMz0vni6PAw, where it encourages making your command "read like sentence (9:53)", and "avoid positional arguments where the order matters (10:35)".
For me it would make more sense having `status checkConnectivity --key KEY --roster ROSTER`. Flag arguments also forces you to describe what they are (not sure what is `private.toml` actually..).

Reply:
This was just a thought for future implementation of CLIs (starts to be a lot of them in cothority).

Reply:
Hmm - I'll follow along whatever you decide, but not before I gave my own opinion ;)

For me, flags are optional. Things that are needed for the command should be part of the arguments. So I always cringe if I have to type

```
bcadmin create --roster public.toml
```

Because the `create` command needs the roster in all cases. Where a flag like `--interval .5s` makes sense to me, as this is not absolutely needed.

Reply:
> Things that are needed for the command should be part of the arguments

I totally agree on this clever principle, which is nice and actually work for commands that have one needed argument, or two if there is a logic (like `cp SOURCE DESTINATION`). 

Otherwise I won't remember the order: is it the roster then the key? or is it the key then the roster? If you don't know it by hearth then you must use `-h`. In my case I don't mind, I know I must provide a key and a roster so, as long as I provide `--roster` and `--key`, my cli is happy. But I am not 100% convinced by myself, so don't change it.

It is actually not easy to find an example of a cli with multiple positional arguments, apart from `cp, mv, ...`.

Reply:
Well you have the firewall `ufw` which takes sometimes more than 2 arguments. But, yes, I have to do `ufw -h` every time I want to use those. But I'm not sure I wouldn't need to if there were flags.

What I try to do in general is to start with the most global, long-term arguments first, then the more specific ones. So for a `roster`, a `config`, and a `key`, this would be the order, as the `roster` is the most global one, then comes my `config`, and finally there is a `key`.

But in general having the CLIs being something like spoken language is nice. Then you just need to avoid the French-based sentence order, else all is off again ;)